apply/golden_file_test.go

```diff
@@ -29,6 +30,7 @@ func TestIntegration(t *testing.T) {
 	for _, tc := range testCases {
 		t.Run(tc.fileName, func(t *testing.T) {
 			a := assert.New(t)
+			r := require.New(t)
```

Comment:
huh, seems like this is what we'd want to be using in general rather than assert